### ISSUE TITLE
[Perception] Scene 4

### ISSUE THREAD

**Issue by jade-mc** [w3c/wai-people-use-web-videos#91]
Scene 4, suggest re-ordering, For example, so that screenreaders can convey what an image is to a user who cannot see it, each image needs to have a text alternative.

**Comment by nitedog** [comment]
Changed as suggested.
